enzyme/Enzyme/InstructionDerivatives.td

```diff
@@ -110,9 +115,22 @@ def : CallPattern<(Op $x),
 def : CallPattern<(Op $x),
                   ["tan", "tanf", "tanl"],
                   [(FMul (DiffeRet<"">), (FAdd (ConstantFP<"1.0"> $x), (FMul(Call<(SameFunc), [ReadNone,NoUnwind]> $x):$c, $c)))]>;
-
-// TODO replace with intrinsic call to exp
+def : CallPattern<(Op $x, $y),
+                  ["remainder"],
+                  [
+                    (DiffeRet<"">),
+                    (FMul (DiffeRet<"">), (Intrinsic<"round", ["orig->getOperand(0)->getType()"]> (FDiv $x, $y)))
```

Comment:
I think this should be -dret * round(x/y).  that is to say -- shouldn't there be a minus sign here?

Reply:
good catch, lost it when replacing constantFP<"-1"> by differet